docs/connector-development/cdk-python/README.md

```diff
@@ -1,16 +1,19 @@
 # Connector Development Kit (Python)
 
+    :::info
+    Singer Taps are no longer supported and contributions of this type will not be accepted by the Airbyte team.
```

Comment:
```suggestion
    The Airbyte team will no longer accept contributions of Singer-based connectors into the Airbyte repo. However, you can still build and use Singer-based connectors in Airbyte using the CDK.
```

Reply:
I don't think this should be so high up in the docs -- we're giving Singer more of a central spot than it needs imo. Can it go in the bottom of this section?

Reply:
> What is the concern around it being a 'central' spot?

it seemed "funny" that the very first thing we say about our own CDK in our own docs is a statement about Singer. We shouldn't be so competitor focused, but rather just focus on our own product.

I like Marcos' suggestion of embedding this note in-line with where we say the CDK supports singer taps

Reply:
Cool I'll move forward with that!